### PR TITLE
Chore: check env files 832

### LINKED ISSUE #832: Check `.env` files for different networks
check that local instance of each network is working properly

- [x] Add libre Mainnet  and libre Testnet 
- [x] Check existing .env files

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolve #832 

### Steps to test

1. Check the examples of env files

#### CheckList

- [ ] The content is adequate
- [ ] I Ran a spell check

### CODE REVIEW

File: .env.mainnet [hunk 1]
```diff
@@ -20,13 +20,13 @@ HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
 
 # hapi
 HAPI_EOS_API_NETWORK_NAME=mainnet
-HAPI_EOS_API_ENDPOINT=https://api.eosio.cr
+HAPI_EOS_API_ENDPOINT=https://api.eosn.io
```
Comment (xavier506) [inline .env.mainnet:23]: lets use our endpoint https://eos.edenia.cloud

File: .env.telos [hunk 2]
```diff
@@ -20,13 +20,13 @@ HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
 
 # hapi
 HAPI_EOS_API_NETWORK_NAME=telos
-HAPI_EOS_API_ENDPOINT=https://telos.eosio.cr
+HAPI_EOS_API_ENDPOINT=https://telos.greymass.com
```
Comment (xavier506) [inline .env.telos:23]: https://telos.edenia.cloud

File: .env.telostestnet [hunk 3]
```diff
@@ -41,25 +41,35 @@ HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
...
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
-REACT_APP_TITLE=EOS Jungle4 Testnet Network Dashboard
+REACT_APP_TITLE=Telos Testnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.edenia.cloud/ipfs/QmX5ENinjMXANryNJWTEWBzxtojtaWzkW1L27YRorrcpcJ
```
Comment (xavier506) [inline .env.telostestnet:63]: we are no longer using IPFS to source images , these are located in the webapp `static` folder now

File: .env.telos [hunk 4]
```diff
@@ -41,17 +41,27 @@ HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
...
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=Telos Mainnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmX5ENinjMXANryNJWTEWBzxtojtaWzkW1L27YRorrcpcJ
```
Comment (xavier506) [inline .env.telos:63]: we are no longer using IPFS for images 

lets set this to : https://eosio.online/images/telos.jpg

File: .env.protontestnet [hunk 5]
```diff
@@ -61,7 +70,7 @@ REACT_APP_HASURA_URL=http://localhost:8585/v1/graphql
 REACT_APP_EOS_API_NETWORK_NAME=proton-testnet
 REACT_APP_EOS_API_NETWORK_LABEL=Proton Testnet
 REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/Qma3imuP4QDJuhAFJKz5AbhgxCLXX7S1V3BrYa8nXNaV9j
```
Comment (xavier506) [inline .env.protontestnet:72]: Use the image hosted in the app

https://eosio.online/images/proton.png

File: .env.protontestnet [hunk 6]
```diff
@@ -45,11 +45,20 @@ HAPI_COINGECKO_API_TOKEN_ID=proton
...
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
 REACT_APP_TITLE=Proton Testnet Network Dashboard
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/Qma3imuP4QDJuhAFJKz5AbhgxCLXX7S1V3BrYa8nXNaV9j
```
Comment (xavier506) [inline .env.protontestnet:62]: Use this image https://eosio.online/images/proton.png

File: .env.proton [hunk 7]
```diff
@@ -45,11 +45,21 @@ HAPI_COINGECKO_API_TOKEN_ID=proton
...
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+
 
 #webapp
 PORT=3000
 REACT_APP_TITLE=Proton Network Dashboard
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
```
Comment (xavier506) [inline .env.proton:62]: We are no longer hosting this on IPFS

use this URL instead : https://eosio.online/images/proton.png

File: .env.mainnet [hunk 8]
```diff
@@ -45,10 +45,19 @@ HAPI_COINGECKO_API_TOKEN_ID=eos
...
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=EOS Mainnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmQ48Sysmfhb9T8Cu3ECNzKCqxTbyuz6n2vWEpTLigmijQ
```
Comment (xavier506) [inline .env.mainnet:63]: Use this image lURL instead
https://eosio.online/images/eos.png

File: .env.libretestnet [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+REACT_APP_FOOTER_LINKS=[{ "text": "Libre Website", "src": "https://libre.org/" },{ "text": "Libre Block Explorer", "src": "https://libre-testnet-explorer.edenia.cloud/" },{"text": "Documentation","src": "https://libre-chain.gitbook.io/"},{"text": "Libre Network Monitor","src": "https://libre-testnet.eosio.online"}]
+REACT_APP_EOS_RATE_LINK=
+REACT_APP_USE_REWARDS=true
+REACT_APP_USE_VOTES=true
+REACT_APP_USE_CPU_BENCHMARK=false
+REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT=true
+REACT_APP_HASURA_URL=http://localhost:8585/v1/graphql
+REACT_APP_EOS_API_NETWORK_NAME=libre-testnet
+REACT_APP_EOS_API_NETWORK_LABEL=Libre Testnet
+REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmcK3TPKZB7fRK5MYYcpUZuZeWpjWEhF1GUY64gGYrXU1K
```
Comment (xavier506) [inline .env.libretestnet:73]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/libre.png

File: .env.libretestnet [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+
+#webapp
+PORT=3000
+REACT_APP_TITLE=Libre Testnet Network Dashboard
+REACT_APP_VERSION=dev
+REACT_APP_NAME=
+REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmcK3TPKZB7fRK5MYYcpUZuZeWpjWEhF1GUY64gGYrXU1K
```
Comment (xavier506) [inline .env.libretestnet:63]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/libre.png

File: .env.libre [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+REACT_APP_FOOTER_LINKS=[{ "text": "Bitcoin Libre Website", "src": "https://libre.org/" },{ "text": "Libre Block Explorer", "src": "https://libre-explorer.edenia.cloud/" },{"text": "Documentation","src": "https://libre-chain.gitbook.io/"},{"text": "Libre Network Monitor","src": "https://libre.eosio.online"}]
+REACT_APP_EOS_RATE_LINK=
+REACT_APP_USE_REWARDS=false
+REACT_APP_USE_VOTES=false
+REACT_APP_USE_CPU_BENCHMARK=false
+REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT=true
+REACT_APP_HASURA_URL=http://localhost:8585/v1/graphql
+REACT_APP_EOS_API_NETWORK_NAME=libre
+REACT_APP_EOS_API_NETWORK_LABEL=Libre Mainnet
+REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmWY6WC6Gy193rKTnHNKzeGruXBGkTQzEVDfwn3j41W235
```
Comment (xavier506) [inline .env.libre:73]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/libre.png

File: .env.libre [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+
+#webapp
+PORT=3000
+REACT_APP_TITLE=Libre Network Dashboard
+REACT_APP_VERSION=dev
+REACT_APP_NAME=$npm_package_name
+REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmQ48Sysmfhb9T8Cu3ECNzKCqxTbyuz6n2vWEpTLigmijQ
```
Comment (xavier506) [inline .env.libre:63]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/libre.png

File: .env.lacchain [hunk 13]
```diff
@@ -43,10 +43,21 @@ HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
...
+HAPI_COINGECKO_API_TOKEN_ID=
+HAPI_REWARDS_TOKEN=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=LACChain Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmS5XzgSEAubehxZVwiHnaLpo3Pv2E5yeEPBhhv6JWSKHE
```
Comment (xavier506) [inline .env.lacchain:63]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/lacchain.jpg

File: .env.lacchain [hunk 14]
```diff
@@ -61,7 +72,7 @@ REACT_APP_EOS_API_NETWORK_NAME=lacchain
 REACT_APP_EOS_API_NETWORK_LABEL=LACChain
 REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmdfvY1gJk688Sxd7iTg6Dbd5VQTv9Mr1M21Fo7nvJapRv
```
Comment (xavier506) [inline .env.lacchain:73]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/lacchain.jpg

File: .env.jungle [hunk 15]
```diff
@@ -45,18 +45,27 @@ HAPI_COINGECKO_API_TOKEN_ID=eos
...
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=EOS Jungle Testnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmfQSA2Qnd6hF1dmnBcTaynp1723PAzY3CdMxeWz8yEdt3
```
Comment (xavier506) [inline .env.jungle:63]: no longer using IPFS for this , use this URL instead

https://eosio.online/images/jungle.jpg

File: .env.telos [hunk 16]
```diff
@@ -61,7 +71,7 @@ REACT_APP_HASURA_URL=http://localhost:8585/v1/graphql
 REACT_APP_EOS_API_NETWORK_NAME=telos
 REACT_APP_EOS_API_NETWORK_LABEL=Telos Mainnet
 REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmX5ENinjMXANryNJWTEWBzxtojtaWzkW1L27YRorrcpcJ
```
Comment (xavier506) [inline .env.telos:73]: please set correct image

File: .env.telostestnet [hunk 17]
```diff
@@ -41,25 +41,35 @@ HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
...
 REACT_APP_USE_REWARDS=true
 REACT_APP_USE_VOTES=true
 REACT_APP_USE_CPU_BENCHMARK=true
 REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT=true
 REACT_APP_HASURA_URL=http://localhost:8080/v1/graphql
-REACT_APP_EOS_API_NETWORK_NAME=telos
-REACT_APP_EOS_API_NETWORK_LABEL=Telos
+REACT_APP_EOS_API_NETWORK_NAME=telos-testnet
+REACT_APP_EOS_API_NETWORK_LABEL=Telos Testnet
 REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.edenia.cloud/ipfs/QmX5ENinjMXANryNJWTEWBzxtojtaWzkW1L27YRorrcpcJ
```
Comment (xavier506) [inline .env.telostestnet:73]: please set correct image

File: .env.waxtestnet [hunk 18]
```diff
@@ -45,11 +45,21 @@ HAPI_COINGECKO_API_TOKEN_ID=wax
...
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
 REACT_APP_TITLE=WAX Testnet Network Dashboard
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
+REACT_APP_NAME=
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmcK3TPKZB7fRK5MYYcpUZuZeWpjWEhF1GUY64gGYrXU1K
```
Comment (xavier506) [inline .env.waxtestnet:63]: please set correct image

**Review by xavier506 — COMMENTED** [review]
Please note we are no longer using the domain name `eosio.cr` we moved all services to `edenia.cloud` please update this domain name wherever necessary

**Review by fvives99 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**